backend/scripts/sync_ac_registry.py

```
@@ -0,0 +1,238 @@
+    for line in body.splitlines():
+        match = _AC_LINE_RE.match(line)
+        if not match:
+            continue
+        ac_id = f"{issue_number}-AC{match.group('num')}"
+        out[ac_id] = _clean_description(match.group("rest"))
+    return out
+
+
+def _fetch_issue_body(issue_number: int) -> str:
+    """Fetch one issue's markdown body via the ``gh`` CLI (network)."""
+    result = subprocess.run(
```

Ruff (0.15.17)

[error] `subprocess` call: check for execution of untrusted input

(S603)